neofetch

```diff
@@ -866,7 +866,7 @@ image_source="auto"
 # CRUX, Crystal Linux, Cucumber, CutefishOS, CuteOS, CyberOS, dahlia, DarkOs, Darwin, Debian,
 # Deepin, DesaOS, Devuan, DietPi, digital UNIX, DracOS, DragonFly, Drauger, Droidian, Elementary,
 # Elive, EncryptOS, EndeavourOS, Endless, Enso, EuroLinux, EvolutionOS, eweOS, Exherbo, Exodia Predator OS,
-# Fedora, FemboyOS, Feren, Finnix, Floflis, FreeBSD, FreeMiNT, Frugalware, Funtoo, GalliumOS,
+# Fedora, FemboyOS, Feren, Finnix, Floflis, FreeBSD, FreeMiNT, Frugalware, Funtoo, Furreto, GalliumOS,
```

Comment:
You mentioned not being sure what this does in the commit; this huge block prints the default config file and this option lets you override the detected OS. This is just so a user knows most of the valid options for what they can override to.

Reply:
ohh so it's like saying what distros are available and such? i get it now

Reply:
FemboyOS :rofl: i gotta get that :rofl:

Reply:
Yep yep. For if you prefer a different distros logo to your own (for example I may override mine to Furreto instead of Gentoo since I think the paw is cute), or for testing things related to ascii art of a distro you're not using.